### PR TITLE
Implement Routing

### PR DESCRIPTION
**Pull request by dominikszczuka**

### CODE REVIEW

**Comment by WiktorMonka** [comment]
Please use correct naming convention for pull request (like first pull request)

File: src/Components/Nav/Nav.tsx [hunk 1]
```diff
@@ -4,20 +4,21 @@ import NavLink from "../NavLink/NavLink";
...
-import { Link } from "constants/types/Link";
+import { LinkElements } from "constants/types/Link";
+
 const style = bemCssModule(NavLinkStyles);
 
 const Nav = () => {
   const { t } = useTranslation();
 
-  const links: Link[] = [
+  const links: LinkElements[] = [
```
Comment (WiktorMonka) [inline src/Components/Nav/Nav.tsx]: Please try to be consistent with interfaces name. The There is an array of string[] not strings[], number[] not numbers[].

File: src/Components/Nav/Nav.tsx [hunk 2]
```diff
@@ -4,20 +4,21 @@ import NavLink from "../NavLink/NavLink";
...
     {
       title: `${t("menu.pos1")}`,
-      to: "/task",
+      to: "/",
     },
     {
       title: `${t("menu.pos2")}`,
```
Comment (WiktorMonka) [inline src/Components/Nav/Nav.tsx]: Please use other naming convention. The naming like pos1, pos2 is not clear and if you want to change order of items there will be a lot of changes.

File: src/navigation/RootNavigator/RootNavigator.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+import HomeScreen from "screens/HomeScreen/HomeScreen";
+import HistoryScreen from "screens/HistoryScreen/HistoryScreen";
+
+const RootNavigator = () => {
+  return (
+    <Router>
+      <Switch>
+        <Route exact path="/" component={HomeScreen} />
+        <Route path="/history" component={HistoryScreen} />
```
Comment (WiktorMonka) [inline src/navigation/RootNavigator/RootNavigator.tsx]: You can add Route for other paths. Something like 404 page

File: src/constants/types/Link.ts [hunk 4]
```diff
@@ -1,4 +1,4 @@
-export interface Link {
-    title:string,
-    to:string,
-}
+export interface LinkElements {
```
Comment (WiktorMonka) [inline src/constants/types/Link.ts]: Please change naming convention. ([2,5,7] => [number,number,number] not [numbers, numbers, numbers]

**Review by WiktorMonka — CHANGES_REQUESTED** [review]
Please read comments

**Comment by WiktorMonka** [comment]
Please dont't merge before we accept cr changes

**Review by MRadek55 — APPROVED** [review]

**Review by WiktorMonka — APPROVED** [review]

**Pull request merged**